lib/ClimaCoreVTK/test/runtests.jl

```diff
@@ -4,8 +4,14 @@ using IntervalSets
 import ClimaCore:
     Geometry, Domains, Meshes, Topologies, Spaces, Fields, Operators
 
+running_CI() = !isempty(get(ENV, "CI", ""))
 
-dir = mktempdir()
+dir = if running_CI()
+    mkdir(joinpath(@__DIR__, "output"))
+else
+    mktempdir()
+end
+mkdir(joinpath(dir, "series"))
```

Comment:
It is sometimes useful to save the results when running the test locally as well. Maybe we could do something like
```suggestion
dir = get(ENV, "CI_OUTPUT_DIR", mktempdir())
```
and then just set the environment variable in the .yml file.

Reply:
Good suggestion, I'll also these for the other artifacts.